src/story/Story.js

```diff
@@ -434,128 +345,102 @@ FORGE.Story.prototype._createScenes = function(config)
 };
 
 /**
- * Internal envent handler for scene load start, updates the group index, re-dispatch scene load start at the story level.
- * @method FORGE.Story#_sceneLoadStart
+ * Internal envent handler for scene load request.
+ * @method FORGE.Story#_sceneLoadRequestHandler
+ * @param  {FORGE.Event} event - The {@link FORGE.Event} emitted by the scene that its load method is requested.
  * @private
- * @param  {FORGE.Event} event - The {@link FORGE.Event} emitted by the scene that starts to load.
  */
-FORGE.Story.prototype._sceneLoadStart = function(event)
+FORGE.Story.prototype._sceneLoadRequestHandler = function(event)
 {
+    var previousScene = this.scene;
+    var nextScene = event.emitter;
+    var time = 0;
+
     //Unload the previous scene
-    if(this._sceneUid !== "" && this._sceneUid !== null)
+    if(previousScene !== null)
     {
-        this.scene.unload();
+        // If the next scene have to be sync with the previous one
+        if(nextScene.sync.indexOf(previousScene.uid) !== -1 && previousScene.media.type === FORGE.MediaType.VIDEO)
+        {
+            time = previousScene.media.displayObject.currentTime;
+        }
+
+        previousScene.unload();
     }
 
-    var scene = event.emitter;
-    this._sceneUid = scene.uid;
+    this._sceneUid = nextScene.uid;
+
+    nextScene.loadStart(time);
+
+    this.log("scene load request");
 
     //The scene has no group so nullify the _groupUid
-    if(scene.hasGroups() === false)
+    if(nextScene.hasGroups() === false)
     {
         this._groupUid = null;
 
         if(this._onGroupChange !== null)
         {
-            this._onGroupChange.dispatch();
+            this._onGroupChange.dispatch({ groupUid: this._groupUid });
         }
 
         if(FORGE.Utils.isTypeOf(this._events.onGroupChange, "ActionEventDispatcher") === true)
         {
             this._events.onGroupChange.dispatch();
         }
+    }
+    else if (nextScene.hasGroups() === true && nextScene.hasGroup(this._groupUid) === false)
+    {
+        this._setGroupUid(nextScene.groups[0].uid);
+    }
 
+    if(this._onSceneLoadRequest !== null)
+    {
+        this._onSceneLoadRequest.dispatch({ sceneUid: this._sceneUid });
     }
-    else if (scene.hasGroups() === true && scene.hasGroup(this._groupUid) === false)
+
+    if(FORGE.Utils.isTypeOf(this._events.onSceneLoadRequest, "ActionEventDispatcher") === true)
     {
-        this._setGroupUid(scene.groups[0].uid);
+        this._events.onSceneLoadRequest.dispatch();
     }
+};
+
+/**
+ * Internal envent handler for scene load start, updates the group index, re-dispatch scene load start at the story level.
+ * @method FORGE.Story#_sceneLoadStartHandler
+ * @private
+ */
+FORGE.Story.prototype._sceneLoadStartHandler = function()
+{
+    this.log("scene load start");
 
     if(this._onSceneLoadStart !== null)
     {
-        this._onSceneLoadStart.dispatch({ uid: scene.uid });
+        this._onSceneLoadStart.dispatch({ sceneUid: this._sceneUid });
```

Comment:
Same here. You also forgot to update `FORGE.PluginManager#_sceneLoadStartHandler` which is still using `did`.